### ISSUE TITLE
why is the loss so high and does not converge even i fine tune a pretrained model on voc?

### ISSUE THREAD

**Issue by birham-red-bd** [balancap/SSD-Tensorflow#287]
The loss when I trained model was decreasing, but it still remains very high, so the detection result is so bad. Then i tried to fine tune the pretrained model，the loss is still very high and did not converge, but the detection result is good. why is that?

**Comment by HiKapok** [comment]
@tf24-karatzhong maybe you can try [this one](https://github.com/HiKapok/SSD.TensorFlow), if you still cannot figure out all the problems in this re-implemtation

**Comment by asjmasjm** [comment]
I have encountered the same problem and the loss result is much more stable using @HiKapok's framework to train either VOC or other self-prepared datasets. Currently the loss under @HiKapok's is around 15 and validate accuracy is 0.82 while the loss swings from 1 to as high as 500 (even nan) under this framework. @HiKapok, thanks for your contribution.
